editor/src/error.rs

```diff
@@ -0,0 +1,29 @@
+use fyrox::{
+    core::{algebra::Vector2, color::Color},
+    gui::{
+        brush::Brush, text::TextBuilder, widget::WidgetBuilder, window::WindowBuilder,
+        UserInterface,
+    },
+};
+
+pub struct ErrorViewer;
+
+impl ErrorViewer {

```

Comment:
Just print an error into the log via `Log::err("message")` and it will automagically appear in the editor's log.